autoload/neomake/makers/ft/erlang.vim

```diff
@@ -5,8 +5,68 @@ endfunction
 
 function! neomake#makers#ft#erlang#erlc() abort
     return {
+        \ 'args': function('neomake#makers#ft#erlang#GlobPaths'),
         \ 'errorformat':
             \ '%W%f:%l: Warning: %m,' .
             \ '%E%f:%l: %m'
         \ }
 endfunction
+
+function! neomake#makers#ft#erlang#GlobPaths() abort
+    " Find project root directory.
+    let rebar_config = neomake#utils#FindGlobFile('rebar.config')
+    if !empty(rebar_config)
+        let root = fnamemodify(rebar_config, ':h')
+    else
+        " At least try with CWD
+        let root = getcwd()
+    endif
+    let root = fnamemodify(root, ':p')
+    let build_dir = root . '_build'
+    let ebins = []
+    if isdirectory(build_dir)
+        " Pick the rebar3 profile to use
+        let profile = 'default'
+        if expand('%') =~# '_SUITE.erl$'
+            let profile = 'test'
+        endif
+        let ebins += neomake#makers#ft#erlang#Glob(build_dir . '/' . profile . '/lib/*/ebin')
+        let target_dir = build_dir . '/neomake'
+    else
+        let target_dir = tempname()
```

Comment:
This will use a new tempdir every time.
Is the target dir helpful when running multiple times, i.e. does it act as a cache?
If not, you could just use `neomake#compat#dev_null` (https://github.com/neomake/neomake/pull/1917).

Reply:
It doesn't, `target_dir` is only set not to clutter CWD / the project directory.
I  tried using `/dev/null`, but since it's a file, not a directory, it generates an unnecessary `|| /dev/null/z.bea#: error writing file: not a directory` in the location list.
I think a global / per-buffer target dir would make the most sense.

Reply:
The error when using /dev/null could be filtered out by the errorformat.